sentry-sidekiq/lib/sentry/sidekiq/sentry_context_middleware.rb

```diff
@@ -2,14 +2,15 @@
 
 module Sentry
   module Sidekiq
-    class SentryContextMiddleware
+    class SentryContextServerMiddleware
       def call(_worker, job, queue)
         return yield unless Sentry.initialized?
 
         context_filter = Sentry::Sidekiq::ContextFilter.new(job)
 
         Sentry.clone_hub_to_current_thread
         scope = Sentry.get_current_scope
+        scope.set_user(scope.user)
```

Comment:
shouldn't the server middleware get the user from job hash instead of its scope (which would just be `nil`)?

Reply:
That's so true, thanks for catching it!